docs/monitoring.md

```diff
@@ -92,6 +92,8 @@ NGINX Gateway Fabric exports the following metrics:
   - nginx_stale_config. 1 means NGF failed to configure NGINX with the latest version of the configuration, which means
     NGINX is running with a stale version.
   - nginx_last_reload_milliseconds. Duration in milliseconds of NGINX reloads (histogram).
+  - event_batch_processing_milliseconds: Duration in milliseconds of event batch processing (histogram). The time it
```

Comment:
I suggest adding a bit more documentation here.

Duration in milliseconds of event batch processing (histogram), which is the time it takes NGF to process batches of Kubernetes events ((changes to cluster resources). Note that NFG processes events in batches, and while processing the current batch, it accumulates events for the next batch.